### PR TITLE
expression: implement vectorized evaluation for builtinSysDateWithoutFspSig

### PR DESCRIPTION
**Pull request by ekalinin**
PCP #12101

### What problem does this PR solve? 

This PR implements vectorized `builtinSysDateWithoutFspSig`

### What is changed and how it works?

```sh
➜ make vectorized-bench VB_FILE=Time VB_FUNC=builtinSysDateWithoutFspSig
cd ./expression && \
	go test -v -benchmem \
		-bench=BenchmarkVectorizedBuiltinTimeFunc \
		-run=BenchmarkVectorizedBuiltinTimeFunc \
		-args "builtinSysDateWithoutFspSig"
goos: linux
goarch: amd64
pkg: github.com/pingcap/tidb/expression
BenchmarkVectorizedBuiltinTimeFuncGenerated-12    	1000000000	         0.00967 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinSysDateWithoutFspSig-VecBuiltinFunc-12         	  751180	      1467 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinTimeFunc/builtinSysDateWithoutFspSig-NonVecBuiltinFunc-12      	    6912	    177445 ns/op	       0 B/op	       0 allocs/op
PASS
ok  	github.com/pingcap/tidb/expression	2.443s

```

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: expression/builtin_time_vec.go [hunk 1]
```diff
@@ -136,11 +136,28 @@ func (b *builtinFromUnixTime2ArgSig) vecEvalString(input *chunk.Chunk, result *c
...
 }
 
 func (b *builtinSysDateWithoutFspSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	loc := b.ctx.GetSessionVars().Location()
+	nowTs, err := getStmtTimestamp(b.ctx)
+	if err != nil {
+		return err
+	}
```
Comment (Reminiscent) [inline expression/builtin_time_vec.go]: It seems these lines are not used. Can we delete them?

**Comment by Reminiscent** [comment]
Please solve the conflicts, thanks!

File: expression/builtin_time_vec.go (hunk 1 above)
Comment (ekalinin) [reply expression/builtin_time_vec.go]: `nowTs` is used later. To get a `now` value.

**Comment by ekalinin** [comment]
Some related changes #13425

**Pull request merged**